### PR TITLE
Adding astroquery project

### PR DESCRIPTION
**Pull request by bsipocz**
@pllim - let me know if you would like to be a mentor again for this project.

@eteq - you were listed last year, but as I recall it was mostly because of the possibility that the project will include adding MAST. Let me know if you still interested for other archives (or if you have preferences which ones should get the priority).

### CODE REVIEW

**Comment by bsipocz** [comment]
cc @keflavich

File: _projects/2018/astropy/astroquery.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ - https://github.com/astropy/astroquery/issues/226
+mentors:
+ - keflavich
+ - bsipocz
+ - pllim
+initiatives:
+ - GSOC
+tags:
+ - python
+ - astropquery
```
Comment (dpshelio) [inline _projects/2018/astropy/astroquery.md]: Should it say `astroquery`?

File: _projects/2018/astropy/astroquery.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+collaborating_projects:
+ - astropy
+ - astroquery
+---
+
+#### Description
+
+[astroquery](http://astroquery.readthedocs.io/), an astropy
+affiliated package, is a package to place modules that access astronomical data
+that is onlone. The aim of this project will be to implement just such modules,
```
Comment (dpshelio) [inline _projects/2018/astropy/astroquery.md]: `onlone` -> `online`

File: _projects/2018/astropy/astroquery.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+for a variety of astronomical archives that are currently lacking astroquery modules.
+A list of new services that have been suggested by our users
+[can be found here](https://github.com/astropy/astroquery/issues?q=is%3Aissue+is%3Aopen+label%3A%22New+Service%22) and
```
Comment (dpshelio) [inline _projects/2018/astropy/astroquery.md]: `here` on websites links is [not good practice](https://webaccess.berkeley.edu/ask-pecan/click-here), better to say in the issues of the project.

**Comment by bsipocz** [comment]
thanks @dpshelio

**Pull request merged**